Operations Documents/Effective Meetings.md

```diff
@@ -0,0 +1,46 @@
+# Effective meetings and group work
+
+To increase flexibility in how and where employees at Clef work, we agree on a shared set of constraints for group work and meetings which are strictly upheld by everyone on the team. 
+
+## Meeting Ettiquette
+
+### Meetings start on time
+
+If you're leading a meeting, it's your responsibility to start the meeting on time. If you're attending a meeting, you are responsible for showing up on time.
+
+### Meetings happen in regular hours
+
+All team members should make themselves available for face-to-face meetings between 10:30am-1:00p Pacific Time, as long as they are scheduled at least 24 hours in advance. If a meeting needs to happen outside of these hours, the meeting organizer should scheduled it a week in advance. 
+
+### Meetings should happen over video
```

Comment:
All meetings should happen over video? What about in-person meetings? I think this is probably a holdover from copying this out of the remote work doc